### ISSUE TITLE
Right frame doesn't hide when clicking "I know this term well"

### ISSUE THREAD

**Issue by jzohrab** [HugoFara/lwt#61]
The code in `insert_word_ignore.php` implies that the right frame should disappear, which would be nice:

```
    window.parent.document.getElementById('frame-l').focus();
    window.parent.setTimeout('cClick()', 100);
```

A few things wrong with this:

* the setTimeout should be the function name, not a string, `setTimeout(cClick, 100);`  (at least, this is a strong recommendation from the js community, even though using a string works :-)  ref https://stackoverflow.com/questions/6081560/is-there-ever-a-good-reason-to-pass-a-string-to-settimeout )
* cClick currently fails with this console message:

```
overlib_mini.js:189 Uncaught TypeError: Cannot read properties of null (reading 'style')
    at hideObject (overlib_mini.js:189:27)
    at eval (eval at runHook (overlib_mini.js:309:48), <anonymous>:1:1)
    at runHook (overlib_mini.js:309:48)
    at cClick (overlib_mini.js:45:32)
```

**Comment by jzohrab** [comment]
Fix done in #62 that makes this LWT much smoother for me to use.

**Comment by HugoFara** [comment]
Merged your PR, I figured out what was the issue. The structure ``window.parent.setTimeout('cClick()', 100);`` seems a bit wrong, but makes sense in the context:
* We have multiple frames.
* A *child* frame wants to close the overing div in *parent* window, for instance when you change word status.
* So it should call ``cClick`` from parent context, using a timeout.
  * Either you call ``setTimeout(window.parent.cClick, time)``
  * You can also use a string to be evaluated in the context, so ``window.parent.setTimeout('cClick', time)``.

As I already merged some of the frames, things are easier today, but the code may stay a bit ugly while things are not standard.
